CTP/shell/init_path/init.sh

```diff
@@ -1122,7 +1109,26 @@ function xkill_java_windows {
 
 
 function get_win_service_pid {
-    wmic PROCESS WHERE \( Name != \'wmic.exe\' and CommandLine LIKE \'%service.Server%\' \)  get processid | grep -v ProcessId | sed 's/ //g'
+    wmic PROCESS WHERE "(
+        Name = 'System Idle Process' or 
+        Name = 'System' or 
+        Name = 'smss.exe' or 
+        Name = 'csrss.exe' or 
+        Name = 'wininit.exe' or 
+        Name = 'winlogon.exe' or 
+        Name = 'services.exe' or 
+        Name = 'lsass.exe' or 
+        Name = 'svchost.exe' or 
+        Name = 'conhost.exe' or 
+        Name = 'WmiPrvSE.exe' or
+        Name = 'taskhostex.exe' or 
+        Name = 'dwm.exe' or 
+        Name = 'rdpclip.exe'
```

Comment:
(in my opinion)
How about adding to 'explorer.exe'??
That is the Windows OS default process.
if it is destroyed then auto-restart the process or the OS does strange.

Reply:
killing explorer.exe will end the taskbar and desktop environment, significantly impacting the user's ability to interact with the system.
 the review is reflected.